app/models/job.rb

```diff
@@ -31,10 +31,27 @@ class Job
 
   attr_accessor :arguments
   attr_accessor :id
-  attr_accessor :min_nodes
   attr_accessor :partition
   attr_accessor :script
   attr_accessor :state
+  attr_accessor :args
```

Comment:
We already have `attr_accessor arguments` set for `Job`.  I prefer `arguments`.

Reply:
Opps, completely missed it

https://github.com/openflighthpc/flight-scheduler-controller/pull/2/commits/5e3568c41a14271540d5d886a61f6cb4ff21a7d3